src/client/Microsoft.Identity.Client.Extensions.Msal/Accessors/LinuxKeyRingAccessor.cs

```diff
@@ -103,11 +104,14 @@ public void Clear()
                 try
                 {
                     GError err = (GError)Marshal.PtrToStructure(error, typeof(GError));
-                    _logger.LogError($"An error was encountered while clearing secret from keyring in the {nameof(Storage)} domain:'{err.Domain}' code:'{err.Code}' message:'{err.Message}'");
+                    throw new InteropException(
```

Comment:
Note that this exception is used for the MacOS accessor. Most APIs either ignore exceptions.